sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -86,7 +86,12 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
    */
   override def visitResetConfiguration(
       ctx: ResetConfigurationContext): LogicalPlan = withOrigin(ctx) {
-    ResetCommand
+    val raw = remainder(ctx.RESET.getSymbol)
+    if (raw.nonEmpty) {
+      ResetCommand(Some(raw.trim))
+    } else {
+      ResetCommand(None)
+    }
```

Comment:
Have been fixed typo except here, @gatorsmile could you please explain a little more? Thanks!